### ISSUE TITLE
Could you provide category classifier models? 

### ISSUE THREAD

**Issue by lennartvandeguchte** [DataSciencePolimi/NewsAnalyzer#1]
I'm struggling to get this pipeline working because the category classifier models are not provided. Is it possible to publish these pre-trained models?

**Comment by GiovanniBrena** [comment]
Hi, I've published the pre-trained models into "category_classifier" folder.

However it is important that you know that those models have been trained on articles dating back about a year ago. Surely the classification accuracy is linked to the topics discussed in that period and I suggest you to generate the model again.

**Comment by lennartvandeguchte** [comment]
I understand, thanks a lot! What did you use as your training data? Did you scrape articles and only used articles from which the category could be extracted from the URL as ground truth? Or did you use an external dataset?

**Comment by GiovanniBrena** [comment]
Yes, I used about 100k url-labeled articles scraped using the pipeline in order to infer categories on not labeled articles.

**Comment by lennartvandeguchte** [comment]
If found a bug in twitter_pipeline/news_extractor.py. On line 17 the statement 'status.user.lang' always returns None and therefore no user data is extracted. 

Furthermore, I have now scraped only url-labeled articles using the list of sources you provided. However, I extracted about 10k articles and only 123 instances of politics articles. How did you extract 100k articles? Did you use more sources? Or did you run the script multiple times over a longer period of time? I'm building a politics/non-politics classifier and therefore I want to have as much political related articles as possible. 

EDIT: Nevermind, I fixed it.
